lte/gateway/c/core/oai/tasks/amf/amf_fsm.cpp

```diff
@@ -157,6 +157,9 @@ void create_state_matrix() {
   Update_ue_state_matrix(
       DEREGISTERED, STATE_PDU_SESSION_RELEASE_COMPLETE, SESSION_NULL,
       DEREGISTERED, SESSION_NULL, "PDU_Release");
+  Update_ue_state_matrix(
```

Comment:
leave a single empty line to make formatting consistent with the other calls above.